package.json

```diff
@@ -1,105 +1,105 @@
 {
-  "name": "beet",
```

Comment:
intentional change in this file?

Reply:
@grctest

Reply:
Sorry, I think I save & committed this file changing just rc1 to rc2, but I had a different spacing enabled switching between projects I'm working on. I'll revert the change to improve readability.